src/lib/Scenes/Home/Components/Sales/__tests__/__snapshots__/index-tests.tsx.snap

```diff
@@ -397,12 +398,12 @@ exports[`looks correct when rendered 1`] = `
                 style={
                   Array [
                     Object {
-                      "height": 201.5,
+                      "height": -36.58,
```

Comment:
The layout doesn't seem to propagate down the tree to the `SaleListItem`s. Currently thinking of ways to test this in a nicer way.

Reply:
Maybe my refactor suggestion also fixes this as a side-effect?